pyproject.toml

```diff
@@ -9,8 +9,10 @@ license = {file = "LICENSE"}
 readme = "README.md"
 requires-python = ">= 3.8.1"
 dependencies = [
-    "presidio-analyzer",
-    "presidio-anonymizer",
+    "presidio-analyzer==2.2.354",
+    "presidio-anonymizer==2.2.354",
+    "numpy==1.26.4",
```

Comment:
Pinning to the exact version (e.g. for numpy) may cause incompatibilities in the dev env -- it's probably more flexible to pin to a max version